src/ExportHelper.php

```diff
@@ -127,4 +127,23 @@ public static function addQuotesToJsonKeys(string $input): string
 
         return $output;
     }
+
+    /**
+     * @param array<mixed, mixed> $mapping
+     */
+    public static function removeTypesInMappingKeys(array &$mapping): void
+    {
+        $final = [];
+        foreach ($mapping as $k => &$v) {
+            if (is_string($k)) {
+                // only $oid and $date has to be preserved
```

Comment:
> only $oid and $date has to be preserved

preco? trocha mi to naznacuje, ze to asi rozbije incremental fetching, ten nevidim ani v testoch

`$oid and $date` nie su nicim specialne, `$oid` je len default sort, k sa nemylim

Reply:
<img width="799" alt="Snímek obrazovky 2022-11-29 v 15 25 34" src="https://user-images.githubusercontent.com/27994036/204555228-ce071bce-8cce-41bb-8448-5577d4fc5637.png">
<img width="833" alt="Snímek obrazovky 2022-11-29 v 15 25 49" src="https://user-images.githubusercontent.com/27994036/204555234-a755d60c-3bf8-4992-89ef-33558d377136.png">

Reply:
Upravím to podle té docky

Reply:
:+1: ok, vdaka za vysvetlenie